tools/install-monorepo.sh

```diff
@@ -45,7 +45,7 @@ fi
 info "Checking if Homebrew is installed..."
 if ! command -v brew &>/dev/null; then
 	info "Installing Homebrew"
-	/bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"
+	/usr/bin/env bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"
```

Comment:
Another one of these
```suggestion
	bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"
```